### PR TITLE
Show NFT Gallery scrollbar on the edge

### PR DESCRIPTION
**Pull request by michalszorad**
Before
Screenshot 2023-03-10 at 14 22 10

After
Screenshot 2023-03-10 at 14 21 28

### CODE REVIEW

File: src/Nfts/ImageGallery/ImageGallery.tsx [hunk 1]
```diff
@@ -237,6 +237,7 @@ function GalleryList<T>({renderItem, ...rest}: FlashListProps<T> & {renderItem:
           <Spacer height={ROW_SPACING} />
         </View>
       )}
+      contentContainerStyle={{paddingHorizontal: CONTAINER_HORIZONTAL_PADDING}}
```
Comment (michalszorad) [inline src/Nfts/ImageGallery/ImageGallery.tsx:238]: Actual fix

File: src/Nfts/Nfts.tsx [hunk 2]
```diff
@@ -194,7 +194,6 @@ const styles = StyleSheet.create({
     justifyContent: 'center',
   },
   galleryContainer: {
-    paddingHorizontal: 16,
```
Comment (michalszorad) [inline src/Nfts/Nfts.tsx:197]: Actual fix

File: src/Nfts/ImageGallery/ImageGallery.tsx [hunk 3]
```diff
@@ -203,9 +203,9 @@ const styles = StyleSheet.create({
     alignItems: 'center',
     justifyContent: 'center',
   },
-  imageContainer: {},
```
Comment (michalszorad) [inline src/Nfts/ImageGallery/ImageGallery.tsx:206]: Minor refactoring

**Review by banklesss — APPROVED** [review]

File: src/Nfts/ImageGallery/ImageGallery.tsx [hunk 4]
```diff
@@ -169,7 +169,7 @@ function SkeletonImagePlaceholder({text}: {text?: string}) {
     )
 
   return (
-    <View style={styles.imageContainer}>
+    <View>
```
Comment (stackchain) [inline src/Nfts/ImageGallery/ImageGallery.tsx]: is it needed?

**Comment by wolverineks** [comment]
overlayColor is android only, have you checked ios?

**Comment by wolverineks** [comment]
due to corner radius, the images are not filling the container
look into `resizeMode`

File: src/Nfts/ImageGallery/ImageGallery.tsx (hunk 4 above)
Comment (michalszorad) [reply src/Nfts/ImageGallery/ImageGallery.tsx]: Not needed, I've just removed the extra `View`

**Comment by michalszorad** [comment]
@wolverineks ios seems to work fine

**Comment by michalszorad** [comment]
@wolverineks, what do you mean by "images are not filling the container"? The border-radius is according to designs. Please correct me if I'm wrong

**Comment by wolverineks** [comment]
> @wolverineks, what do you mean by "images are not filling the container"? The border-radius is according to designs. Please correct me if I'm wrong

do some of the images have borders?

**Review by stackchain — APPROVED** [review]

**Comment by michalszorad** [comment]
@wolverineks there is only a border-radius in the designs, no other borders

**Review by wolverineks — APPROVED** [review]

**Pull request merged**